staging/src/k8s.io/apiserver/pkg/registry/rest/resttest/resttest.go

```diff
@@ -917,11 +919,15 @@ func (t *Tester) testDeleteWithResourceVersion(obj runtime.Object, createFn Crea
 
 	foo := obj.DeepCopyObject()
 	t.setObjectMeta(foo, t.namer(1))
-	objectMeta := t.getObjectMetaOrFail(foo)
-	objectMeta.SetResourceVersion("RV0000")
 	if err := createFn(ctx, foo); err != nil {
 		t.Errorf("unexpected error: %v", err)
 	}
+	newObj, err := getFn(ctx, foo)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	objectMeta := t.getObjectMetaOrFail(newObj)
+
 	opts.Preconditions = metav1.NewRVDeletionPrecondition("RV1111").Preconditions
```

Comment:
Thanks for fixing the test! For better readability, could we use something else instead of "RV1111" here? Since we removed "RV0000", it is not obvious where "RV1111" is coming from. I am thinking something like objectMeta.GetResourceVersion() - 1

Reply:
@jingyih 👌 Thanks for your review! Please check again.